### PR TITLE
compiling flagser, flagser-count with MANY_VERTICES to fix overflows …

### PR DESCRIPTION
**Pull request by flomlo**
see PR #66 ; everything on the code the same, just different git branch.

### CODE REVIEW

File: pyflagser/tests/test_flagser.py [hunk 1]
```diff
@@ -246,3 +246,10 @@ def test_higher_coefficients():
     x = np.random.random((5, 5))
     np.fill_diagonal(x, 0.)
     flagser_weighted(x, coeff=3)
+
+
+def test_really_huge_graphs():
```
Comment (ulupo) [inline pyflagser/tests/test_flagser.py:251]:
```suggestion
def test_huge_graphs():
```

:)

**Review by ulupo — COMMENTED** [review]
Just a boring suggestion :)

**Comment by ulupo** [comment]
@MonkeyBreaker just waiting for your final blessing on this!

**Comment by MonkeyBreaker** [comment]
The changes seems good, I see that the flag was also added to `flagser_count` version, I think it would be a good idea to also add a test for this one. `flagser` and `flagser_count` are not exactly the same thing, even if most of their backend is the same.

Do you think it's worth it @ulupo to add a test ? @flomlo would you be able to add a test also for `flagser_count` version ?

**Comment by flomlo** [comment]
Mhm, I'ld say that this is overtesting, as the bug arises both in `flagser` and `flagser_count` in the shared section of the code, that is, the flag complex generation. Unless there are some MAJOR changes in `flagser` it will test exactly the same thing.

(I really hate writing test code, as well)

**Comment by MonkeyBreaker** [comment]
You're right, looking again at the code, I cannot see where it could produce a different/unexpected behavior.

(^^, after loosing a lot of time on a different project because we didn't have regression test in place at the time, I lost a lot of hair on a bug that I thought was fixed. That's why I'm now like "add test, add test" ...)

Then, @ulupo all green for me.

**Comment by ulupo** [comment]
I forgot why this was not merged ages ago already. @MonkeyBreaker is everything still green on your side?

**Comment by MonkeyBreaker** [comment]
Let me just test if this fixes #65 for last message posted, 5 minutes :)

**Comment by MonkeyBreaker** [comment]
With this fix, I can run without any exception thrown !
Thank you again @flomlo 
@ulupo Ok for the merge for me

**Pull request merged**